### ISSUE TITLE
Dynamic table names in Query

### ISSUE THREAD

**Issue by Fyb3roptik** [go-pg/pg#745]
I have views that are based on id's. I am trying to run a query to grab the view data, but the table name ? is not being updated with the variable.

```
DB().Query(&plays, `SELECT * FROM contest_?_?_leaderboard ORDER BY rank ASC`, c.Id, tier.Id)
```

**Comment by vmihailenco** [comment]
How formatted query looks like? So far I don't see what is wrong.

**Comment by Fyb3roptik** [comment]
This however does work. So if someone else runs across this, this is how you get it working:

```golang
view_name := fmt.Sprintf("contest_%d_%d_leaderboard", c.Id, tier.Id)
DB().Query(&plays, `SELECT * FROM ? LIMIT ? OFFSET ? ORDER BY rank ASC`, view_name, perpage, perpage * page)
```

**Comment by siteshen** [comment]
@Fyb3roptik It doesn't work because `view_name` is not a sql parameter. You can try this:
```go
sql :=  fmt.Sprintf(`SELECT * FROM contest_%d_%d_leaderboard ORDER BY rank ASC LIMIT ? OFFSET ?`, a, b)
DB().Query(&plays, sql, perpage, perpage * page)
```

**Comment by Fyb3roptik** [comment]
That works!!! But is the first issue a bug? Should the ?'s in:

```sql
DB().Query(&plays, `SELECT * FROM contest_?_?_leaderboard LIMIT ? OFFSET ? ORDER BY rank ASC`, c.Id, tier.Id, perpage, perpage * page)
```

Shouldn't the ?'s in the table name get filled in? If not, then maybe document this somewhere? Thanks again for a great ORM

**Comment by vmihailenco** [comment]
What happens here is that go-pg parses `?_` as param with name `_`, because we support named params like `?foo ?bar` etc. And since there is no param with such name it simply restores `?_` and processes other placeholders. I am not sure what is the right fix/behavior here.

**Comment by vmihailenco** [comment]
As a workaround you can use `fmt.Sprintf` or named params:

```
params := struct{
    params1, param2 int
}{1, 2}
db.Query(&plays, `SELECT ?param1_param2`)
```

**Comment by Fyb3roptik** [comment]
Ah! ok I will do that. Thanks again for a great ORM!

**Comment by vmihailenco** [comment]
This should be fixed in v6.7.1. Please try to upgrade.
